src/components/templates/Jazzbar/Jazzbar.tsx

```diff
@@ -7,6 +7,8 @@ interface PropsType {
   venue?: Venue;
 }
 
+// @debt This whole venue component looks ugly. There are a lot of places to improve it, but
+// you can see it yourself, when you start understanding the way it works
```

Comment:
Could we please stick to more 'objective' comments for these sorts of `@debt` things. I'd like to ensure our codebase/culture doesn't get too 'aggressive' or feel 'unfriendly' for newcomers/etc.

Perhaps just:

```suggestion
// @debt This venue component needs a cleanup to align with our desired patterns
```